### DISCUSSION TITLE
Picklist and OrderList Virtual Scroll Support

### DISCUSSION THREAD

**Discussion by thang1160** [primefaces/community#774] [PrimeNG] [answered]
In the document, the picklist doesn't have a virtual scroller, so with my current data (>30,000), it took quite a long time to render. Which, is pretty bad for UX

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Orderlist has virtualScroll support, we haven't implemented the virtualScroll for picklist yet. We've plans to implement it also to the picklist in the future releases.
